source/_components/lutron_caseta.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Lutron_Caseta"
+description: "Instructions how to use Lutron_Caseta devices with Home Assistant."
+date: 2017-01-28 13:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lutron.png
+ha_category: Hub
+featured: False
+ha_release: 0.40
```

Comment:
Update version to 0.41 or later if that's more likely to be the version it's merged into.